astropy/_erfa/tests/test_erfa.py

```diff
@@ -189,27 +192,27 @@ def test_matrix_in():
                                              -.1809906511146821008])
     np.testing.assert_allclose(astrom['bpn'], np.eye(3))
 
-    # first make sure it *fails* if we mess with the input orders
-    pvmatbad = np.roll(pvmat.ravel(), 1).reshape((2, 3))
-    astrombad = erfa.apcs13(jd1, jd2, pvmatbad)
-    assert not np.allclose(astrombad['em'], 1.010428384373318379)
-
-    pvmatarr = np.array([pvmat]*3)
-    astrom2 = erfa.apcs13(jd1, jd2, pvmatarr)
-    assert astrom2.shape == (3,)
-    np.testing.assert_allclose(astrom2['em'], 1.010428384373318379)
-
-    # try striding of the input array to make non-contiguous
-    pvmatarr = np.array([pvmat]*9)[::3]
-    astrom3 = erfa.apcs13(jd1, jd2, pvmatarr)
-    assert astrom3.shape == (3,)
-    np.testing.assert_allclose(astrom3['em'], 1.010428384373318379)
-
-    # try fortran-order
-    pvmatarr = np.array([pvmat]*3, order='F')
-    astrom4 = erfa.apcs13(jd1, jd2, pvmatarr)
-    assert astrom4.shape == (3,)
-    np.testing.assert_allclose(astrom4['em'], 1.010428384373318379)
+    # # first make sure it *fails* if we mess with the input orders
+    # pvmatbad = np.roll(pvmat.ravel(), 1).reshape((2, 3))
+    # astrombad = erfa.apcs13(jd1, jd2, pvmatbad)
+    # assert not np.allclose(astrombad['em'], 1.010428384373318379)
+
+    # pvmatarr = np.array([pvmat]*3)
+    # astrom2 = erfa.apcs13(jd1, jd2, pvmatarr)
+    # assert astrom2.shape == (3,)
+    # np.testing.assert_allclose(astrom2['em'], 1.010428384373318379)
+
+    # # try striding of the input array to make non-contiguous
+    # pvmatarr = np.array([pvmat]*9)[::3]
+    # astrom3 = erfa.apcs13(jd1, jd2, pvmatarr)
+    # assert astrom3.shape == (3,)
+    # np.testing.assert_allclose(astrom3['em'], 1.010428384373318379)
+
+    # # try fortran-order
+    # pvmatarr = np.array([pvmat]*3, order='F')
+    # astrom4 = erfa.apcs13(jd1, jd2, pvmatarr)
+    # assert astrom4.shape == (3,)
+    # np.testing.assert_allclose(astrom4['em'], 1.010428384373318379)
```

Comment:
Was this temporarily commented out?

Reply:
Some of those tests no longer make sense, since the pv vectors are not input as `(..., 2, 3)` arrays. I'll keep what does work still.